packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -103,6 +126,12 @@ export function specTest(
         runner = it.only;
       } else if (!WEB_SPEC_TEST_FILTER(tags)) {
         runner = it.skip;
+      } else if (usePersistence && tags.indexOf('no-lru') !== -1) {
+        runner = (fullname: string, fn: () => {}) => {
+          const skipComment = comment || '';
+          // tslint:disable-next-line:no-console
+          console.log('skipping "' + fullName + '"\n\t' + skipComment);
```

Comment:
This is a little different from how we handle other "skipped" tests. If you think it adds value to print the comment during the test run, you could just call `console.log` here and then use it.skip as your runner. Alternatively, you could rely on these comments as they show up in the test definition and just use `it.skip`.

Reply:
Done.